DisCatSharp/Clients/DiscordClient.Dispatch.cs

```diff
@@ -890,11 +876,7 @@ internal async Task OnChannelUpdateEventAsync(DiscordChannel channel)
 
 			channelNew.PermissionOverwritesInternal.Clear();
 
-			foreach (var po in channel.PermissionOverwritesInternal)
-			{
-				po.Discord = this;
-				po.ChannelId = channel.Id;
-			}
+			channel.Initialize(this);
```

Comment:
This honestly seems a bit out of place? I feel like it might belong after adding the `PermissionOverWritesInternal` in the next line so that the id of the new channel will be added to the overwrites.

Reply:
imma look into it tmrw